### ISSUE TITLE
Employee Info (Admin Mode) - Ensure every form element has a label

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#261]
Title: WCAG 1.3.1,WCAG 3.3.2: Ensures every form element has a label (#checkbox-468)
Tags: Accessibility, WCAG 1.3.1, WCAG 3.3.2, label

Issue: Ensures every form element has a label (label - https://dequeuniversity.com/rules/axe/3.5/label?application=msftAI)

Target application: ALEKSANDAR BATINIC - Odoo - http://localhost:8069/web#id=9421&action=137&model=hr.employee&view_type=form&cids=1&menu_id=95

Element path: #checkbox-468

Snippet: <input type="checkbox" id="checkbox-468" class="custom-control-input" disabled="">

<img width="1680" alt="Screen Shot 2020-04-27 at 10 39 20 AM" src="https://user-images.githubusercontent.com/31868510/80384841-5fe9dd00-8873-11ea-85e2-699f6e501015.png">

How to fix: 
Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  Element has no title attribute or the title attribute is empty

Environment: Chrome version 80.0.3987.163

====

This accessibility issue was found using Accessibility Insights for Web 2.16.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
